Decky Cloud Save v1.3.3

GedasFX opened the pull request:
<!-- Make sure to include your plugin name below! -->

# Decky Cloud Save v1.3.3

A plugin based on [rclone](https://rclone.org/), which allows users to back-up game saves to the cloud. Supports [OneDrive](https://onedrive.live.com/), [Google Drive](https://drive.google.com/), [Dropbox](https://www.dropbox.com/), and [many more](https://github.com/GedasFX/decky-cloud-save#backend-support)!

**Changes**:

+ Bumped rclone version (1.61.1 to 1.65.1)
~ Minor fix for bisync experimental feature not working (thank you @TJSomething)

Also contains hidden feature from 1.3.2: https://github.com/SteamDeckHomebrew/decky-plugin-database/pull/465. This is a hidden opt in feature that works for some people. Its opt in and doesn't affect the rest of the plugin so I think it can be in, otherwise I can remove it.

## Checklist:

### Developer Checklist

- [x] I am the original author or an authorized maintainer of this plugin.
- [x] I have abided by the licenses of the libraries I am utilizing, including attaching license notices where appropriate.

### Plugin Checklist

- [x] I have verified that my plugin works properly on the Stable and Beta update channels of SteamOS.
- [x] I have verified my plugin is unique or alternatively provides more/alternative functionality to a similar plugin already on the store.

<!-- The following section needs to be modified as yes/no answers by the plugin developer. -->

<!-- Ex: "**Yes/No**: ..." becomes "**Yes**: ..." -->

### Plugin Backend Checklist

- **No**: I am using a custom backend other than Python.
- **No**: I am using a tool or software from a 3rd party FOSS project that does not have it's dependencies [statically linked](https://en.wikipedia.org/wiki/Static_library).
- **Yes**: I am using a custom binary that has all of it's dependencies statically linked.

<!-- The following section is should be modified to fit the conditions for plugin testing found here: https://wiki.deckbrew.xyz/en/plugin-dev/review-and-testing -->

## Testing

<!-- Remove this box for SteamOS Preview testing if you do not use a custom or remote binary, for more info follow the URL in the comment above the testing section. -->
- [ ] Tested on SteamOS Preview Update Channel.

TrainDoctor commented:
Approved for testing store.

TJSomething reviewed (commented):
On more thorough testing, I can't approve this. Sorry I didn't see this earlier.

Decky v1.10.10, Decky CloudSave - 1.3.3-79607de
1. 1 blocking issue found:
    1. Using the expected procedure for enabling bisync makes it impossible to use the UI to disable bisync. It appears that the code is always using the second key in `plugin.properties` to determine if bisync is enabled, which on a clean install will be the `experimental_menu` setting, not the `bisync_enabled` setting.
2. No minor issues found.
3. Other currently installed plugins: None
4. No go for prod

GedasFX commented:
@TJSomething 
1. Fixed the issue, I always tested this with experimental feature, now it looks for things correctly https://github.com/GedasFX/decky-cloud-save/pull/55
2. Updated docs. https://github.com/GedasFX/decky-cloud-save/commit/06830eb7d65948dd50b6b443a5d1bf12979c329b

TJSomething reviewed (approved):
Decky v2.10.10, Decky CloudSave - 1.3.3-4200928

1. No blocking issues found.
2. No non-blocking issues found.
3. Other currently installed plugins: None
4. Approved for prod
